digitalstrategie/templates/base.html

```diff
@@ -13,6 +13,10 @@
         <link rel="icon" type="image/png" sizes="96x96" href="{% static 'images/favicon-96x96.png' %}">
         <link rel="icon" type="image/png" sizes="16x16" href="{% static 'images/favicon-16x16.png' %}">
 
+        {% block meta_tag %}
```

Comment:
a couple of lines above we still have the "static" meta title and empty meta description. Do we want to keep those?
It leads to two `<title>`, `<meta description>` etc.

Reply:
https://github.com/liqd/digitalstrategie/blob/716b845ea8be83cbb81a9a861b7c956827d59aed/digitalstrategie/templates/base.html#L8

Reply:
Oh, yes, this is redundant! I also noticed now, thought, that the <meta name="author" content=""> isn't rendered by the new metatags. Is this relevant to us @fuzzylogic2000 ? Otherwise, we can take out both.

Reply:
Yes, we should take that out! Good catch!